website_portal_medical_insurance_us/security/ir.model.access.csv

```diff
@@ -0,0 +1,5 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_medical_insurance_template_portal,medical.insurance.template.portal,medical_insurance.model_medical_insurance_template,base.group_portal,1,1,1,0
+access_medical_insurance_plan_portal,medical.insurance.plan.portal,medical_insurance.model_medical_insurance_plan,base.group_portal,1,1,1,0
+access_medical_insurance_company_portal,medical.insurance.company.portal,medical_insurance.model_medical_insurance_company,base.group_portal,1,1,1,0
```

Comment:
@lasley this is a little weird haha. So I commented out the domain force rules in the manifest and only have these access records. What I don't get is that after creating a plan as portal user with the insurance company field populated, it's showing access errors when showing `/medical/insurance/plans` after that.

Yet we clearly have access granted to the portal user for medical.insurance.company. in the csv file.
```
Error to render compiling AST
AccessError: (u'The requested operation cannot be completed due to security restrictions. Please contact your system administrator.\n\n(Document type: medical.insurance.company, Operation: read)', None)
Template: website_portal_medical_insurance_us.insurances
Path: /templates/t/t/t[2]/div/t/div/div/div[2]/div[1]/div
Node: <div class="h5" t-field="plan.insurance_company_id.name"/>
```

Reply:
Insurance companies inherit from abstract entity, which delegate inherits from partner. Try adding a domain_force to allow the showing of partners when `create_uid == user.id`